.husky/pre-commit

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```
suggestion: I think it's considered safer to get the location of `bash` from the system `env`.